src/components/offers/applyForm/ApplyForm.js

```diff
@@ -54,8 +60,13 @@ export const ApplyForm = () => {
             title: "Applying"
         })
         Swal.showLoading()
-        offersAPI.applyWithNoUser(id, name, surname, email, files)
-            .then((result) => {
+        let applyPromise
+        if (loggedIn)
+            applyPromise = offersAPI.applyWithUser(id)
+        else
+            applyPromise = offersAPI.applyWithNoUser(id, name, surname, email, files)
```

Comment:
```js
const applyPromise = loggedIn ? offersAPI.applyWithUser(id) : offersAPI.applyWithNoUser(id...)
```
Plus przy okazji jak już przy tym grzebiesz, to możesz zmienić na withSwal, które pojawiło się później